### ISSUE TITLE
need more meaningful requests.ConnectionError exception messages

### ISSUE THREAD

**Issue by billimek** [homedepot/flow#22]
See https://github.com/homedepot/flow/blob/master/flow/artifactstorage/artifactory/artifactory.py#L126-L127

It is currently,

```
        except requests.ConnectionError:
            commons.print_msg(Artifactory.clazz, method, "Request to Artifactory timed out.", "ERROR")
            exit(1)
```

But this will inform the user _Request to Artifactory timed out._ for ANY ConnectionError exception which can mislead on what the actual cause is.

We probably want to change it to something like,

```
        except requests.ConnectionError as e:
            error = str(e)
            commons.print_msg(Artifactory.clazz, method, "Request to Artifactory raised a ConnectionError: " + error, "ERROR")
            exit(1)
```

**Comment by dwood12** [comment]
Does this issue have a tracker story yet?

**Comment by billimek** [comment]
Fixed by #24
